bot.py

```diff
@@ -348,66 +486,63 @@ class EditSelectMenu(discord.ui.Select):
         will be used as the main embed.
         ctx (`discord.ext.commands.Context`): necessary parameter when accesing
         some discord server data. Used by internal methods.
+        update_flag (`bool`): A raised flag to indicate whether
+        a new embed will be initialized or the previous one updated.
     """
 
-    def __init__(self, new_embed, ctx: commands.Context):
-        self.embed = new_embed
-        self.ctx = ctx
-
-        options = [
-            discord.SelectOption(
-                label="Title and Message",
-                description="Set a title and description for the embed",
-            ),
-            discord.SelectOption(
-                label="Add Field", description="Add a field to the embed"
-            ),
-            discord.SelectOption(
-                label="Author", description="Set a author for the embed"
-            ),
-            discord.SelectOption(
-                label="Thumbnail", description="Set a thumbnail for the embed"
-            ),
-            discord.SelectOption(
-                label="Image", description="Set an image for the embed"
-            ),
-            discord.SelectOption(
-                label="Footer", description="Set a footer for the embed"
-            ),
-            discord.SelectOption(
-                label="Color", description="Set a color for the embed"
-            ),
-        ]
+    def __init__(
+        self, new_embed: discord.Embed, ctx: commands.Context, update_flag: bool
+    ):
         super().__init__(
             placeholder="Expand the list to edit the embed's...",
-            options=options,
             min_values=1,
             max_values=1,
+            options=[
+                discord.SelectOption(
+                    label="Title and Message",
+                    description="Set a title and description for the embed",
+                ),
+                discord.SelectOption(
+                    label="Add Field", description="Add a field to the embed"
+                ),
+                discord.SelectOption(
+                    label="Remove Field", description="Remove a field from the embed"
+                ),
```

Comment:
Wydaje mi się, że aby nie przytłoczyć użytkownika nie lepiej by było Zamiast AddField i RemoveField dać po prostu jedną opcję. Field i działałaby ona tak, `Title and Message`. 
Czyli jeśli jakieś wartości były już wpisane to po prostu pojawi się już wypełniony formularz.
To by bardzo ułatwiło testowanie widoku w podglądzie wiadomości (no i łatwe poprawianie błędów)
![image](https://github.com/KNR-PW/discord-bot/assets/45544416/141f91f0-c26d-4fac-89fd-0b6c67db78d0)

Reply:
Poza tym wydaje mi się, że dodawanie wielu pól tak jak na widocznym rysunku chyba nie jest potrzebna.   
Funkcja jest fajna, ale na ten moment chyba byłoby to trochę przekombinowane dla osób, które mają z tego docelowo korzystać

Reply:
Chociaż z drugiej strony trochę żal to usuwać.

Dałoby się po prostu dać `Edit Field: Lista osób` dla każdego z już utworzonych pól?
Aby dało się poprawić już istniejące pole bez pisania wszystkiego jeszcze raz?

Reply:
Myślę, że tak. Tylko trzeba wziąć pod uwagę, że jest możliwość dodania do 25 pół tekstowych. Czy chciałbyś w takim razie, żeby menu tekstowe, które może mieć do 25 pól wyboru było dynamiczne i wraz z dodaniem pola, zwiększało się o wiersz z edycją wybranego pola? W jaki sposób pola mają być usuwane? Polem wyboru, które jest teraz czy czymś na zasadzie, że jeżeli usunie się treść pola i wyśle je, to znika?

Reply:
Jeżeli chodzi o kwestionariusze, które się wypełnia, to sprawdziłem, że maksymalnie może być 5 pól na raz, dlatego przy więcej niż jednym dodatkowym polu tekstu, trzeba by wprowadzić coś typu lista wyboru pola, która prowadzi do edycji zawartości konkretnego pola, ale wtedy gdyby się usunęło np. jakieś środkowe pole, a potem chciało dodać nowe na któreś miejsce, to musiałaby być kolejna lista wyboru, pytająca, gdzie ma być umieszczone nowe pole.
Daj znać jak to widzisz i czy chciałbyś coś takiego, czy zrezygnować z tego.

Reply:
To już byłoby chyba zbyt przekombinowane.

Chyba prościej jest zrezygnować z tego, bo te dodatkowe pola i tak są tylko dodatkiem, a nie kluczową funkcją.   
Pozostawiam ci wolną rękę e tym jak to zrobisz. W sumie jeśli chcesz to możesz to zostawić jak jest i w razie jak uzytkownicy docelowi będą narzekać to wtedy to się poprawi.

Reply:
W takim razie na ten moment rezygnuję z rozbudowy tej funkcji.

Reply:
+ zapomniałem jeszcze dodać śledzenie zmian w dodatkowych polach. Myślę, że ograniczę ich ilość do 5 np.

Reply:
> W kolejnym branchu dodałbym zabezpieczenia, żeby tylko odpowiednie osoby mogły korzystać z komend.

Będę miał pytanie jaki chciałbyś żeby parametr był sprawdzany przy próbie wywołania komendy. W tym momencie tylko administrator serwera może używać komend, ale jest to bardzo "customizowalne". Począwszy od sprawdzania, czy wywołujący posiada jakąś specyficzną rolę, do sprawdzania, czy może korzystać z któregoś z wymienionych parametrów:
![image](https://github.com/KNR-PW/discord-bot/assets/117805804/4fcd4bd4-8231-4ad4-a9ad-4976544c3f8e)

Reply:
Osoba mogąca przeglądać logi serwera lub osoba, która może zarządzać rolami.

Reply:
https://github.com/KNR-PW/discord-bot/pull/19